proto_text.c

```diff
@@ -961,6 +993,20 @@ static int _meta_flag_preparse(token_t *tokens, const size_t ntokens,
     p += 2; \
 }
 
+// TODO: calc bytes remaining in buffer
```

Comment:
ah, should do this before merge.

Reply:
Isn't this already implicitly calculated? since 'p' itself is the 'wbuf' ? And 'wbytes' will later be adjusted based as 'p = resp->wbuf'

Reply:
Think I meant to actually use the bytes remaining to avoid potentially overflowing.